tests/cpp/parser_tests.cpp

```diff
@@ -8517,5 +8517,50 @@ rule module_rule
 	EXPECT_EQ(input_text, driver.getParsedFile().getTextFormatted());
 }
 
+TEST_F(ParserTests,
+ExpressionArrayWorks) {
+	prepareInput(
+R"(
+rule expression_array
+{
+	condition:
+		any of (1, 2, 3)
```

Comment:
Just a note as we have discussed this. The fix for this in upstream is already created: https://github.com/VirusTotal/yara-x/pull/208